app/src/main/java/org/gdg/frisbee/android/fragment/PulseFragment.java

```diff
@@ -139,16 +151,15 @@ public void onNotFound(String key) {
         });
     }
 
-    private void initAdapter(Pulse pulse) {
-        mPulse = pulse;

```

Comment:
Because mPulse wasn't used. It could be converted to a local variable, which wasn't actually used.